web/app.js

```diff
@@ -1457,6 +1460,9 @@ function webViewerInitialized() {
     if ('useonlycsszoom' in hashParams) {
       PDFJS.useOnlyCssZoom = (hashParams['useonlycsszoom'] === 'true');
     }
+    if ('printautorotate' in hashParams) {
```

Comment:
Remove the hash parameter addition. The hash parameters are only used for debugging purposes, not for toggling features. Moreover, using the preferences for this is preferred and quite a few hash parameters have already been removed before.